### DISCUSSION TITLE
Scanning All the repos in an Azure devops (ADO) repo using secrets manager and Amazon

### DISCUSSION THREAD

**Discussion by munntjlx** [praetorian-inc/noseyparker#118] [Ideas]
I have developed (in conjunction with sending results to defect dojo) a series of scripts and processes that enable full scanning of an Azure devops 'organization' as well as retrieving all the repos for said org. In short, it creates a list of targets, then uses that list as a URL to clone using git. Once git clones (locally) we scan with noseyparker, and then output in jsonl results which can be fed to defect dojo. Its not perfect, but minimally might be interesting to folks sentenced to use Azure devops instead of git.  Would there be any interest in my submitting a PR with some of these scripts / etc?

**Comment by bradlarsen** [comment]
@munntjlx Yes, that sounds like a useful integration example. If you create a PR with those scripts added under an `examples/azure-devops` directory, I'll take it. Thanks!
